ssl-config/src/test/java/com/palantir/ssl/SerializationTests.java

```diff
@@ -41,4 +38,25 @@ public void testJsonSerDe() throws JsonParseException, JsonMappingException, Jso
         assertThat(mapper.readValue(mapper.writeValueAsBytes(sslConfig), SslConfiguration.class), is(sslConfig));
     }
 
+    @Test
+    public void testJsonDeserialize() throws IOException {
+        ObjectMapper mapper = ObjectMappers.guavaJdk7();
+        SslConfiguration sslConfig = SslConfiguration.of(
+                TestConstants.CA_TRUST_STORE_PATH,
+                TestConstants.SERVER_KEY_STORE_JKS_PATH,
+                TestConstants.SERVER_KEY_STORE_JKS_PASSWORD);
+
+        assertThat(mapper.readValue(JSON_STRING, SslConfiguration.class), is(sslConfig));
+    }
+
+    private static final String JSON_STRING =
```

Comment:
Test shows that serialization format has remained constant (`"JKS"` String is properly read from JSON and converted to enum)